src/reference/asciidoc/index.adoc

```diff
@@ -5,13 +5,13 @@
 :numbered:
 :icons: font
 :hide-uri-scheme:
-Mark Pollack; Mark Fisher; Oleg Zhurakousky; Dave Syer; Gary Russell; Gunnar Hillert; Artem Bilan; Stéphane Nicoll; Arnaud Cogoluègnes; Jay Bryant
+Authors: Mark Pollack; Mark Fisher; Oleg Zhurakousky; Dave Syer; Gary Russell; Gunnar Hillert; Artem Bilan; Stéphane Nicoll; Arnaud Cogoluègnes; Jay Bryant
```

Comment:
I would prefer without `Authors:` since it is pretty nice on the final page after the tittle.

Hence my point do not have `Version:` label as well. 

It is fully clear what is that in bold on its own line 😄 

But I still would like to see version before Copyright:

![image](https://user-images.githubusercontent.com/1071590/51768566-a8ce4700-20ae-11e9-8f3a-babb6381e5b1.png)


If you all are OK, I can merge with that my polishing

Reply:
If you're comfortable making those changes, that's OK by me. Thanks, Artem.